src/applications/gi/components/search/EligibilityForm.jsx

```diff
@@ -5,13 +5,51 @@ import { showModal, hideModal, eligibilityChange } from '../../actions';
 
 import Dropdown from '../Dropdown';
 import RadioButtons from '../RadioButtons';
+import environment from 'platform/utilities/environment';
 
 export class EligibilityForm extends React.Component {
   constructor(props) {
     super(props);
     this.renderLearnMoreLabel = this.renderLearnMoreLabel.bind(this);
   }
 
+  cumulativeServiceOptions = () => {
+    if (environment.isProduction()) {
+      return [
+        { value: '1.0', label: '36+ months: 100% (includes BASIC)' }, // notice not 1.00
+        { value: '0.9', label: '30 months: 90% (includes BASIC)' },
+        { value: '0.8', label: '24 months: 80% (includes BASIC)' },
+        { value: '0.7', label: '18 months: 70% (excludes BASIC)' },
+        { value: '0.6', label: '12 months: 60% (excludes BASIC)' },
+        { value: '0.5', label: '6 months: 50% (excludes BASIC)' },
+        { value: '0.4', label: '90 days: 40% (excludes BASIC)' },
+        { value: '0.0', label: 'Less than 90 days 0% (excludes BASIC)' },
+        { value: '1.00', label: 'GYSGT Fry Scholarship: 100%' }, // notice not 1.0
+        {
+          value: 'service discharge',
+          label: 'Service-Connected Discharge: 100%',
+        },
+        { value: 'purple heart', label: 'Purple Heart Service: 100%' },
+      ];
+    }
+
+    return [
```

Comment:
because we're 3 months out from releasing this, let's add a comment with the issue id and the launch date for this

Reply:
PR came from https://github.com/department-of-veterans-affairs/vets.gov-team/issues/18517

There will be a future story to enable this

Reply:
https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/vets.gov-team/18639